src/components/IndividualReviewPage.jsx

```diff
@@ -20,6 +21,7 @@ const IndividualReviewPage = () => {
       ) : (
```

Comment:
Needs a little bit of styling to prevent comments showing on the left hand side at smaller screen sizes, or is this intentional?